main.yml

```diff
@@ -741,3 +742,19 @@ vulnerability:
       - id: systems
         title: Systems
   source_repo: https://github.com/RedHatInsights/vulnerability-ui
+
+ros:
```

Comment:
I want to keep this config as alphabetized as possible. Can you please move this new config to be between `rhel-sw` and `ruledev`?

Reply:
surely, fixed the order and indentation as well.